packages/wallets/src/styles/fonts.scss

```diff
@@ -0,0 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Sans:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&display=swap');
```

Comment:
@adrienne-deriv Do we use all of the variations? 🤔 We can only add the ones we need to reduce the load time I believe 🤔